Source/Tooltips/Main.lua

```diff
@@ -20,61 +20,53 @@ function Auctionator.Tooltip.ShowTipWithPricing(tooltipFrame, itemId, itemCount)
     countString = "|cFFAAAAFF x" .. itemCount .. "|r"
   end
 
-  local auctionPrice = Auctionator.Database.GetPrice(itemId)
+  local auctionPrice = Auctionator.Database.GetPrice(itemKey)
   if auctionPrice ~= nil then
     auctionPrice = auctionPrice * (showStackPrices and itemCount or 1)
   end
+  
+  local vendorPrice = 0;
+  local cannotAuction = 0;
 
-  local
-    name,
-    link,
-    rarity,
-    level,
-    minLevel,
-    itemType,
-    itemSubType,
-    stackCount,
-    equipLocation,
-    icon,
-    sellPrice,
-    classID,
-    unused,
-    cannotAuction = GetItemInfo(itemId);
-
-  -- TODO Listen to GET_ITEM_INFO_RECEIVED to get info for non-cached items.
-  if name~=nil then
-    local vendorPrice = sellPrice * (showStackPrices and itemCount or 1)
-
-    tooltipFrame:AddDoubleLine("ItemID", itemId)
-
-    Auctionator.Tooltip.AddVendorTip(tooltipFrame, vendorPrice, countString)
-    Auctionator.Tooltip.AddAuctionTip(tooltipFrame, auctionPrice, countString, cannotAuction)
+  if string.sub(itemKey,1,1) == "p" then
+    if auctionPrice ~= nil then
+      Auctionator.Utilities.Message("Pet has AH price "..math.floor(auctionPrice/10000).."g "..math.floor((auctionPrice%10000)/100).."s");
+    end
+  else
+    local _, _, _, _, _, _, _, _, _, _, sellPrice, _, _, cannotAuctionTmp = GetItemInfo(itemKey);
+    cannotAuction = cannotAuctionTmp;
+    vendorPrice = sellPrice * (showStackPrices and itemCount or 1);
+  end
 
-    -- TODO Disenchant price; still need to figure out d/e tables...
+  tooltipFrame:AddDoubleLine("ItemID", itemKey)
```

Comment:
Wrap this in an if debug statement (so we don't need to remember to remove it prior to shipping and so that we can still see it when we're in debug mode).

Reply:
We've already had people comment on it being in the alphas ;)